ui/packages/codice-icons/codice.font.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    'files': [
+        './codice-icons/*.svg'
+    ],
+    'fontName': 'codice-icons',
+    'classPrefix': 'codice-',
+    'baseSelector': '.codice',
```

Comment:
✏️ We might want to use a different base selector than `codice` in case `codice` is eventually useful in other contexts in the future 

`cf` for `codice fonts`? `class="cf cf-map-marker"`? Other input would be appreciated.

Reply:
i like cf better - went ahead and updated 👍